uberfire-simple-docks/uberfire-simple-docks-client/src/main/resources/org/uberfire/client/resources/css/docks.css

```diff
@@ -1,78 +1,124 @@
-.dock {
-    background-color: #e3e3e3;
-    border-style: solid;
-    border-width: 1px;
+.sideDockItem {
+    padding: 5px 5px !important;
+    width: 16px;
+    height: 20px;
 }
 
-.westDockInnerPanel {
-    position: absolute;
-    bottom: 0;
-    transform:rotate(-90deg);
-    transform-origin: top left;
-    width: 800px;
-    margin-right: 2px;
+.singleDockItem {
+    height: 100%;
+    width: 17px;
+    display: table;
+    background-image: linear-gradient(to bottom,#E6E6E6, #FFF) !important;
 }
 
-.eastDockInnerPanel {
-    position: absolute;
-    top: 0;
-    transform:rotate(90deg);
-    transform-origin: bottom left;
-    margin-left: 3px;
-    width: 800px;
+.singleDockItemIcon {
+    display: table-cell !important;
+    vertical-align: middle;
 }
 
-.southDockInnerPanel {
+.hideElement {
+    display: none;
+}
+
+.gradientTopBottom {
+    -moz-box-shadow: rgba(255, 255, 255, 0.2) 0px 1px 0px 0px inset, rgba(0, 0, 0, 0.0470588) 0px 1px 2px 0px;
+    -webkit-box-shadow: rgba(255, 255, 255, 0.2) 0px 1px 0px 0px inset, rgba(0, 0, 0, 0.0470588) 0px 1px 2px 0px;
+    background: rgb(245, 245, 245) linear-gradient(rgb(255, 255, 255), rgb(230, 230, 230)) repeat-x;
+    border-image-outset: 0px;
+    border-image-repeat: stretch;
+    border-image-slice: 100%;
+    border-image-source: none;
+    border-image-width: 1;
+    border: 1px solid rgba(0, 0, 0, 0.0980392);
+    border-top-width: 0px;
+    border-bottom-color: rgb(179, 179, 179);
+    box-shadow: rgba(255, 255, 255, 0.2) 0px 1px 0px 0px inset, rgba(0, 0, 0, 0.0470588) 0px 1px 2px 0px;
+    box-sizing: border-box;
+    color: rgb(51, 51, 51);
+    text-shadow: rgba(255, 255, 255, 0.74902) 0px 1px 1px;
+}
+
+.gradientBottomTop {
+    -moz-box-shadow: rgba(0, 0, 0, 0.0470588) 0px 1px 0px 0px inset, rgba(255, 255, 255, 0.2) 0px 1px 2px 0px;
+    -webkit-box-shadow: rgba(0, 0, 0, 0.0470588) 0px 1px 0px 0px inset, rgba(255, 255, 255, 0.2) 0px 1px 2px 0px;
+    background: rgb(245, 245, 245) linear-gradient(rgb(230, 230, 230), rgb(255, 255, 255)) repeat-x;
+    border-image-outset: 0px;
+    border-image-repeat: stretch;
+    border-image-slice: 100%;
+    border-image-source: none;
+    border-image-width: 1;
+    border: 1px solid rgba(0, 0, 0, 0.0980392);
+    border-top-width: 0px;
+    border-bottom-color: rgb(179, 179, 179);
+    box-shadow: rgba(255, 255, 255, 0.2) 0px 1px 0px 0px inset, rgba(0, 0, 0, 0.0470588) 0px 1px 2px 0px;
+    box-sizing: border-box;
+    color: rgb(51, 51, 51);
+    text-shadow: rgba(255, 255, 255, 0.74902) 0px 1px 1px;
```

Comment:
probably lot's of those properties aren't needed.. like border-image-source: none;